src/plugins/footnotes/index.ts

```diff
@@ -227,6 +294,50 @@ export default () => {
               )
             }
           }
+          if (can.editArticle) {
+            if (isInTableElement) {
```

Comment:
What is the importance of this anyway? As far as I can see it doesn't make a difference.

Reply:
I'm using this specifically for cited footnotes because the type `schema.nodes.footnotes` is shared with the regular footnotes (those outside the table). Removing this check is causing issues with the deletion process. I will replace it with `if (tableElement)` within the table notes pull request. This adjustment is necessary for passing `tableElement` inside the widget to use it for deleting inlineFootnotes.